content/docs/command-reference/get-url.md

```diff
@@ -39,7 +39,7 @@ DVC supports several types of (local or) remote locations (protocols):
 | `local` | Local path     | `/path/to/local/data`                      |
 | `s3`    | Amazon S3      | `s3://mybucket/data`                       |
 | `gs`    | Google Storage | `gs://mybucket/data`                       |
-| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data`     |
+| `ssh`   | SSH server     | `ssh://user@example.com/path/to/data`      |
```

Comment:
do we need the trailing space?

Reply:
sounds good! and we can elaborate when we have a separate page per remote I guess

Reply:
Yes. It's all formatted by Prettier this way.